mantle/kola/tests/misc/selinux.go

```diff
@@ -118,25 +113,9 @@ func getSelinuxBooleanState(c cluster.TestCluster, m platform.Machine, seBool st
 	return boolState, nil
 }
 
-// SelinuxEnforce checks that some basic things work after `setenforce 1`
-func SelinuxEnforce(c cluster.TestCluster) {
-	cmdList := []cmdCheckOutput{
-		{"getenforce", true, "Enforcing"},
-		{"sudo setenforce 0", false, ""},
-		{"getenforce", true, "Permissive"},
-		{"sudo setenforce 1", false, ""},
-		{"getenforce", true, "Enforcing"},
-		{"systemctl --no-pager is-active system.slice", true, "active"},
-		{"sudo cp /etc/selinux/config{,.old}", false, ""},
-		{"sudo sed -i 's/SELINUX=permissive/SELINUX=enforcing/' /etc/selinux/config", false, ""},
```

Comment:
I guess one thing we don't do in the other test is verify we can change from enforcing to permissive mode. Not sure if this is important or not. The other test is a non-exclusive test so I don't know if it would be smart to do that there.

cc @jlebon

Reply:
I think it's worth testing, yeah. We could still get rid of this test here and convert the non-exclusive test to an exclusive one? Draining mantle tests into external tests is still nice.

I'm not sure what the `system.slice` check is doing here. We can probably drop that in the conversion. Also the `sed` bit dates back from CL days I think where SELinux wasn't enforcing by default? But it'd make sense to test the opposite (that you can change to permissive mode via `/etc/selinux/config`, reboot, and verify we're in permissive mode).